seqr/views/apis/auth_api_tests.py

```diff
@@ -68,6 +69,48 @@ def test_login_view(self):
         response_json = response.json()
         self.assertTrue(response_json['success'])
 
+        # send login request with a privileged account (data manage or superuser) while Google auth is not enabled
+        req_values = {
+            'email': 'test_data_manager@test.com',
+            'password': 'not-a-password'
+        }
+        response = self.client.post(url, content_type='application/json',
+                                    data=json.dumps(req_values))
+        self.assertEqual(response.status_code, 401)
+        self.assertEqual(response.reason_phrase, 'Invalid credentials')
+
+        req_values = {
+            'email': 'test_superuser@test.com',
+            'password': 'not-a-password'
+        }
+        response = self.client.post(url, content_type='application/json',
+                                    data=json.dumps(req_values))
+        self.assertEqual(response.status_code, 401)
+        self.assertEqual(response.reason_phrase, 'Invalid credentials')
+
+    @mock.patch('seqr.views.apis.auth_api.SOCIAL_AUTH_GOOGLE_OAUTH2_KEY', 'test_key')
+    def test_login_view_with_google(self):
+        url = reverse(login_view)
+
+        # send login request with a privileged account (data manage or superuser) while Google auth is enabled
```

Comment:
can you also add a test here to ensure that non-priveleged users CAN successfully log in when google auth is enabled? Testing any non-proveleged user is fine, you only need one